clients/drcachesim/tests/offline-tlb_simulator_v2p.templatex

```diff
@@ -84,8 +84,8 @@ Core #3 \(1 thread\(s\)\)
     Local miss rate:                 90.48%
     Child hits:                     *33[,\.]?002
     Total miss rate:                  0.06%
-#elif defined(X86) && defined(X64)
-ERROR: failed to initialize analyzer: Directory setup failed: Failed sanity checks for thread log file .*/drmemtrace.threadsig.aarch64/raw/drmemtrace.threadsig..*.raw.gz: Architecture mismatch: trace recorded on aarch64 but tools built for x86_64
+#elif (defined(X86) && defined(X64)) || defined(ARM)
+ERROR: failed to initialize analyzer: Directory setup failed: Failed sanity checks for thread log file .*/drmemtrace.threadsig.aarch64/raw/drmemtrace.threadsig..*.raw.gz: Architecture mismatch: trace recorded on aarch64 but tools built for .*
```

Comment:
Perhaps it would be nicer to also test the arch in the message. Maybe just duplicate the line for ARM? Same in other files.

Reply:
Done.